### PR TITLE
avoid re-build intel-sgx-ssl and re-download openssl tarball

### PR DESCRIPTION
**Pull request by YangLiang3**

### CODE REVIEW

**Review by haosanzi — APPROVED** [review]

File: rats-tls/cmake/SGXSSL.cmake [hunk 1]
```diff
@@ -44,8 +40,15 @@ set(_make_script "${CMAKE_CURRENT_SOURCE_DIR}/intel_sgxssl_make.sh")
...
     file(WRITE "${_make_script}"
 "#!/bin/sh
-cd ${INTEL_SGXSSL_SRC}/Linux
-make
+if [ ! -d ${INTEL_SGXSSL_LIB_PATH} ]; then
+        cd ${INTEL_SGXSSL_SRC}/Linux;
+        make -j$(nproc)
+fi;
+cd ${INTEL_SGXSSL_LIB_PATH}
+if [ ! -e  libsgx_usgxssl.a ]; then
```
Comment (jiazhang0) [inline rats-tls/cmake/SGXSSL.cmake:48]: plz remove the unnecessary spaces.

File: rats-tls/cmake/SGXSSL.cmake [hunk 2]
```diff
@@ -44,8 +40,15 @@ set(_make_script "${CMAKE_CURRENT_SOURCE_DIR}/intel_sgxssl_make.sh")
...
+        cd ${INTEL_SGXSSL_SRC}/Linux;
+        make -j$(nproc)
+fi;
```
Comment (jiazhang0) [inline rats-tls/cmake/SGXSSL.cmake:51]: Plz combine the above conditions into a single one.

**Review by jiazhang0 — CHANGES_REQUESTED** [review]

File: rats-tls/cmake/SGXSSL.cmake (hunk 1 above)
Comment (YangLiang3) [reply rats-tls/cmake/SGXSSL.cmake:48]: if [ || ] doesn't work

File: rats-tls/cmake/SGXSSL.cmake (hunk 2 above)
Comment (YangLiang3) [reply rats-tls/cmake/SGXSSL.cmake:51]: if [ || ] doesn't work

Comment (jiazhang0) [reply rats-tls/cmake/SGXSSL.cmake:49]: Use '-o'. `if [ ! -d ${INTEL_SGXSSL_LIB_PATH} -o ! -e ${INTEL_SGXSSL_LIB_PATH}/libsgx_usgxssl.a ]; then`

**Review by haosanzi — APPROVED** [review]

**Review by jiazhang0 — APPROVED** [review]

**Pull request merged**